url_shortener/shorten_app/static/css/main.css

```diff
@@ -0,0 +1,58 @@
+@import url('https://fonts.googleapis.com/css2?family=Balsamiq+Sans:wght@700&family=Montserrat:wght@400;500&display=swap');
+
+body {
+    background-color: rgb(131, 182, 248);
+    font-family: 'Montserrat', Arial, Helvetica;
+    margin-top: 50px;
+}
+
+h1 {
+    font-family: 'Balsamiq Sans', Arial, Helvetica;
+}
+
+* { 
+    text-align: center; 
+    color: white;
+}
+
+#enter_url {
+    margin-top: 100px;
+    color: black;
+}
+
+.submit_btn {
+    padding: 10px 20px;
+    background-color: rgb(150, 145, 145);
+    border-style: none;
+    border-radius: 2px;
+    margin-top: 20px;
+    cursor: pointer;
+}
+
+.submit_btn:hover {
+    font-weight: bold;
+}
+
+#newUrlDiv {
+    margin-top: 100px;
+}
+
+#result {
+    padding: 30px;
+    color: blue;
+    text-decoration: underline;
+    background-color: white;
+    border-radius: 8px;
+    font-weight: bold;
+}
+
+#another {
+    text-decoration: none;
+}
+
+footer { 
+    position: fixed; 
+    bottom: 0; 
+    width: 100%; 
+    border-top: 2px solid white;
+}
```

Comment:
lovely styling, I can always tell a project apart when it's you guys. The style is very unique, clean and user friendly. I would possibly set text colour of input field as black because for some reason when you type a url in atm the text is white and blends in with background of input box.